content/open-tech-will-save-us/otwsu.toml

```diff
@@ -0,0 +1,270 @@
+* **Hubert|Uhoreg Chathi** will present on the state of encryption in Matrix, and more broadly
+*Why the PGP model is insufficient, what properties we want from encryption (e.g. why we want perfect forward secrecy), how the newer protocols (OTR, double-ratchet, megolm) give us some of the properties we want, and then what's coming in the future (MLS, and what that gives us)*
+"""
+
+[[episodes]]
+code=7
+youtube="xJ2Tjc1Aj0A"
+date="2020-10-14"
+summary="""
+* **Holger** and **Karissa**, representing [DeltaChat](https://delta.chat/en/), and
+**Matthew** from Matrix, will hold
+*robust discussions* on the subject of Open Tech: will it save us? Surely Open
+Tech can be used for all kinds of bad things? They will also discuss how Matrix 
+and Delta Chat's approaches differ technically and organisationally.
+* **Dendrite**, a next generation Matrix homeserver written in Go, has entered
+*beta*. Join **Kegan**, from the team, to find out the state of the project and why
+we're all excited about it again.
+* **David Jacovkis** ([@dvdjaco:matrix.org](https://matrix.to/#/@dvdjaco:matrix.org))
+from femProcomuns (<https://femprocomuns.coop>)
+will present <https://meet.coop>, a videoconf service operated by coops. He'll discuss
+the challenges and opportunities of this approach for the sustainability of Free and
+Open Source projects.
+"""
+
+[[episodes]]
+code=6
+youtube="AGs4ZC24Q1Y"
+date="2020-09-09"
+summary="""
+* **Ag3m**, from [La Quadrature du Net](https://www.laquadrature.net) will join us to present _"Some thoughts on moderation and censorship in a decentralised world"_ .  
+    As a member at LQDN for three years and a volunteer for more than that, Ag3m cofounded a queer feminist hackerspace in Paris called _le Reset_.
+* **Sean DuBois** ([Sean-Der](https://github.com/Sean-Der),  [@sean-der:matrix.org](https://matrix.to/#/@sean-der:matrix.org)), WebRTC-knower and author of  [WebRTC for the Curious](https://webrtcforthecurious.com) will discuss his recent work in the space.
+* **Damir Jelić** presented the latest news on the Matrix Rust SDK
+"""
+
+[[episodes]]
+code=5
+youtube="tJ8tthkVAOQ"
+date="2020-08-12"
+summary="""
+*  **Jonathan Beri, known Maker of Things** ( [@jberi@toot.cafe](https://toot.cafe/@jberi)) will be presenting _Making it easier to make Things: WebAssembly and the Internet of Things_  
+    WebAssembly is moving beyond the browser - but is it ready for IoT apps and tiny embedded devices? Yes...ish. In this talk, learn about the state of running Wasm on embedded devices and what's left to solve.
+*  **Matthew Petry from Pine64** ( [@fire219:matrix.org](https://matrix.to/#/@fire219:matrix.org) ) presents his experience with running Synapse on RockPro64 cluster, plus some thoughts on his time with Pine64!
+*  **Nad from Element** dives into the recent rebrand, in which he worked to combine three brands into one, all while working to maintain a strong identity - highly recommended for Design and Product nerds fans
```

Comment:
What is with these double spaces? Also should we use `-` as the list item syntax to make it a little easier to see what's the list syntax and what is emphasis?